src/components/ContentHighlights/HighlightStepper/SelectContentSelectionStatus.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import { useContextSelector } from 'use-context-selector';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, DataTableContext } from '@edx/paragon';
+
+import { ContentHighlightsContext } from '../ContentHighlightsContext';
+
+const SelectContentSelectionStatus = ({ className }) => {
+  const { toggleAllRowsSelected } = useContext(DataTableContext);
+  const currentSelectedRowsCount = useContextSelector(
```

Comment:
[note] Test coverage will be added in a future PR here.

Reply:
Leaving open as a reminder to complete